README.md

```diff
@@ -1 +1,28 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+> Welcome to my personal portfolio, feel free to look around.
+
+
```

Comment:
Add the screenshot of the portfolio here

Reply:
Thank you Shaher for your review, do you mean my portfolio or a template screenshot?

Reply:
Should I use my portfolio screenshot or Figma screenshot?

Reply:
You should add your portfolio screenshot so any user can see your work through README file.

Reply:
Ok great! Thank you.